sources/packages/backend/apps/api/src/route-controllers/user/user.institutions.controller.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Patch,
+  UnprocessableEntityException,
+} from "@nestjs/common";
+import { BCeIDService, UserService } from "../../services";
+import BaseController from "../BaseController";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import { BCeIDDetailsAPIOutDTO } from "./models/bceid-account.dto";
+import { InstitutionUserAPIOutDTO } from "./models/institution-user.dto";
+import { InstitutionUserPersistAPIInDTO } from "./models/institution-user-persist.dto";
+import { BCeIDAccountsAPIOutDTO } from "./models/bceid-accounts.dto";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, AllowInactiveUser } from "../../auth/decorators";
+import { ApiTags, ApiUnprocessableEntityResponse } from "@nestjs/swagger";
+import { BCeIDAccountTypeCodes } from "../../services/bceid/bceid.models";
+import { UserControllerService } from "..";
+
+@AllowAuthorizedParty(AuthorizedParties.institution)
+@Controller("users")
+@ApiTags("users")
+export class UserInstitutionsController extends BaseController {
+  constructor(
+    private readonly service: UserService,
+    private readonly bceidService: BCeIDService,
+    private readonly userControllerService: UserControllerService,
+  ) {
+    super();
+  }
+
+  @AllowInactiveUser()
+  @Get("bceid-account")
+  async getBCeID(
+    @UserToken() userToken: IUserToken,
+  ): Promise<BCeIDDetailsAPIOutDTO | null> {
+    const account = await this.bceidService.getAccountDetails(
+      userToken.idp_user_name,
+      // TODO: To be changed to allow basic BCeID sign in.
```

Comment:
Please remove the TODO.
This method is used during institution creation when an business BCeID creates its own institution.
It must allow only `BCeIDAccountTypeCodes.Business`.
Basic BCeID institution are created by the Ministry.